documentation/en/actor-events-api.md

```diff
@@ -0,0 +1,434 @@
+# Actor Events and Lotus APIs
+
+* [Background](#background)
+* [ActorEvent structure](#actorevent-structure)
+* [Querying Lotus for ActorEvents](#querying-lotus-for-actorevents)
+* [Retrieving events from message receipts](#retrieving-events-from-message-receipts)
+* [Current builtin actor event schemas](#current-builtin-actor-event-schemas)
+  * [Verified registry actor events](#verified-registry-actor-events)
+    * [Verifier balance](#verifier-balance)
+    * [Allocation](#allocation)
+    * [Allocation removed](#allocation-removed)
+    * [Claim](#claim)
+    * [Claim updated](#claim-updated)
+    * [Claim removed](#claim-removed)
+  * [Market actor events](#market-actor-events)
+    * [Deal published](#deal-published)
+    * [Deal activated](#deal-activated)
+    * [Deal terminated](#deal-terminated)
+    * [Deal completed](#deal-completed)
+  * [Miner actor events](#miner-actor-events)
+    * [Sector precommitted](#sector-precommitted)
+    * [Sector activated](#sector-activated)
+    * [Sector updated](#sector-updated)
+    * [Sector terminated](#sector-terminated)
+
+## Background
+
+Actor events are a fire-and-forget mechanism for actors in Filecoin to signal events that occur during execution of their methods to external observers. The events themselves are not stored on chain, but are emitted by the actor and may be stored in a node's event store. They are intended to be used by tooling and applications that need to observe and react to events that occur within the chain.
+
+The FEVM already has this capability and these events for builtin actors have been added to support a range of new features, starting at network version 22 with a focus on some information gaps for consumers of data onboarding metrics due to the introduction of Direct Data Onboarding (DDO), plus some additional events related to data onboarding, deal lifecycles, sector lifecycles, and DataCap activity. Additional events are expected to be added in the future to support other features and use cases.
+
+Builtin actor events share basic similarities to the existing events emitted by the FEVM, but are structured differently to reflect Filecoin-specific concerns. There are also new APIs in Lotus to support querying for these events that bear some similarities to the existing FEVM `Eth*` APIs for querying events but are unique to builtin actors.
```

Comment:
```suggestion
Built-in actor events share basic similarities to the existing events emitted by FEVM-based user actors but are structured differently to reflect Filecoin-specific concerns. There are also new APIs in Lotus to support querying for these events that bear some similarities to the existing FEVM `Eth*` APIs for querying events but are unique to builtin actors.
```

Reply:
> but are unique to builtin actors.

is this true now that we went for the generic approach with GetActorEvents & SubscribeActorEvents -> which supports *both* builtin actors and user smart contract events?

Reply:
yeah, revisiting some of this with fresh, and more informed eyes, I see a bunch of stuff that's not _quite_ right. Have reworked this para and some other stuff around it